load_sample_data.sh

```diff
@@ -17,10 +17,10 @@ PGPASSWORD=deploy pg_dump -s -t image -U deploy -d openledger -h localhost -p 54
 # Load sample data
 PGPASSWORD=deploy psql -U deploy -d openledger -h localhost -p 5432 -c "INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false), (now(), 'behance', 'Behance', 'https://www.behance.net', false);"
 PGPASSWORD=deploy psql -U deploy -d openledger -h localhost -p 5433 <<EOF
+ALTER TABLE image RENAME TO image_view;
+ALTER TABLE image_view ADD COLUMN standardized_popularity double precision;
 CREATE TABLE content_provider(provider_identifier varchar(50), provider_name varchar(250), created_on timestamp, domain_name varchar(500), filter_content boolean, notes text); INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false), (now(), 'behance', 'Behance', 'https://www.behance.net', false);
-\copy image (id,created_on,updated_on,identifier,provider,source,foreign_identifier,foreign_landing_url,url,thumbnail,width,height,filesize,license,license_version,creator,creator_url,title,tags_list,last_synced_with_source,removed_from_source,meta_data,tags,watermarked,view_count) from './sample_data/sample_data.csv' with csv header
-CREATE TABLE image_normalized_popularity(identifier uuid, normalized_popularity double precision);
-\copy image_normalized_popularity from './sample_data/sample_popularity_data.csv' with csv header;
+\copy image_view (id,created_on,updated_on,identifier,provider,source,foreign_identifier,foreign_landing_url,url,thumbnail,width,height,filesize,license,license_version,creator,creator_url,title,tags_list,last_synced_with_source,removed_from_source,meta_data,tags,watermarked,view_count,standardized_popularity) from './sample_data/sample_data.csv' with csv header
```

Comment:
This is out of sync with the current `image_view` schema, but not in a way that invalidates tests of the current ingestion strategy (as far as I can tell).  But, you might consider syncing it up to reduce cruft.